rpc/net.go

```diff
@@ -196,17 +169,12 @@ func (s *NetService) takeOffer(multiaddr, offerID string, providesAmount float64
 	skm.OfferID = offerID
 	skm.ProvidedAmount = providesAmount
 
-	if err = s.net.Initiate(who, skm, swapState); err != nil {
+	if err = s.net.Initiate(peer.AddrInfo{ID: who}, skm, swapState); err != nil {
```

Comment:
`(peer.AddrInfo{ID: who}`

this works? interesting

Reply:
Yeah, it works.  libp2p does it in a bunch of places in their code:
https://github.com/libp2p/go-libp2p/blob/master/p2p/host/routed/routed.go#L205